test/touch.html

```diff
@@ -121,6 +131,16 @@
           }, 1000);
         });
 
+        it('should not preventDefault on touch-end after normal tap', done => {
+          makeSoloTouchEvent('touchstart', null, target);
+
+          setTimeout(() => {
+            const e = makeSoloTouchEvent('touchend', null, target);
+            expect(e.defaultPrevented).to.be.false;
+            done();
+          }, 100);
```

Comment:
why `100`?

Reply:
Is it an imitation of normal tap for 1 microsecond?

Reply:
Yes a synthetic tap emulation, actually anything lesser than the timeout that fires the long tap (contextmenu)

Reply:
Would be nice to document these in the source code, so we could avoid these kinds of questions and confusion in the future as well.